friendlist/src/main/java/com/bytro/friendlist/rest/controller/FriendRequestController.java

```diff
@@ -31,4 +35,10 @@ public BaseResponse<Void> acceptRejectFriendRequest(
             @RequestBody AcceptRejectFriendRequest acceptRejectFriendRequest) {
         return friendRequestHandler.acceptRejectFriendRequest(acceptRejectFriendRequest);
     }
+    @Operation(summary = "List of friend request")
+    @GetMapping("/friend-requests")
```

Comment:
we can take this userId in the path variable and page and size in request param

Reply:
at the end, we will take user-id from the token so as of now we can continue with this